rpc/core/blocks.go

```diff
@@ -333,6 +332,10 @@ func EncodeDataRootTuple(height uint64, dataRoot [32]byte) ([]byte, error) {
 	return append(paddedHeight, dataRoot[:]...), nil
 }
 
+// dataCommitmentBlocksLimit The maximum number of blocks to be used to create a data commitment.
+// It's a local parameter to protect the API from creating unnecessarily large commitments.
+const dataCommitmentBlocksLimit = 10_000 // ~33 hours of blocks assuming 12-second blocks.
```

Comment:
why does the PR description say 4 hours but this line says 33 hours?

Reply:
i'll update